src/ui/pattern-player/pattern-player.vue

```diff
@@ -298,10 +336,16 @@
 				border-right: 1px solid #ddd;
 				text-align: center;
 				position: relative;
+                overflow: visible;
+                padding: 0;
 
 				&.has-changes {
 					background-color: #fbe8d0;
 				}
+
+                &.is-triplet {
+                    color: #0000ff;
```

Comment:
Not working for me, all strokes stay blue, whether triplets or not. The rule is overridden : 
<img width="346" height="52" alt="Screenshot 2025-11-27 at 20 13 47" src="https://github.com/user-attachments/assets/4d57397d-a6b8-4eb3-91d2-392f2b41e489" />

This rule is overriding it :
<img width="348" height="79" alt="Screenshot 2025-11-27 at 20 13 53" src="https://github.com/user-attachments/assets/5a8ccb3b-4c8c-4a8f-89c6-0f0f247e4895" />